test/get-predeployed-accounts.test.ts

```diff
@@ -0,0 +1,15 @@
+import { expect } from "chai";
+import { starknet } from "hardhat"
+
+describe("Devnet restart", function() {
+    this.timeout(900_000);
```

Comment:
See how timeout is specified in other tests.

Reply:
I import TIMEOUT from constant and replace 900_000 with it [48fde15](https://github.com/Shard-Labs/starknet-hardhat-example/pull/66/commits/48fde15de89778456fb896baec7b05938e12a5b2)